aikau/src/main/resources/alfresco/lists/Paginator.js

```diff
@@ -755,6 +767,99 @@ define(["dojo/_base/declare",
          }
          delete this.__deferredLoadedDocumentData;
          this._controlsLoaded.resolve();
+      },
+      
+      /**
+       *  Handles initialization and necessary updates of the page selection menu items when the popup is opened.
+       * 
+       *  @instance
+       *  @param {object} position the position of the selection menu popup when opened
+       */
+      checkAndUpdatePageSelectionMenu : function alfresco_lists_Paginator__checkAndUpdatePageSelectionMenu() {
+          var firstPage, lastPreloadedPage, pageStart, pageEnd, i, label, menuItem;
+
+          // only need to act if not initialized yet or change occurred
+          firstPage = 0;
+          lastPreloadedPage = -1;
+          
+          // Delete the (obsolete) previous page selector group contents...
+          array.forEach(this.pageSelectorGroup.getChildren(), function(widget, index, arr) {
+              var virtPageEnd;
+              
+              // need to destroy every existing menu item if documentsPerPage has changed
+              // or state hasn't been fully initialized yet
+              if (this._pageSelectionDocsPerPage === undefined
+                      || this._pageSelectionDocsPerPage !== this.documentsPerPage
```

Comment:
JSHint is complaining about the line breaks here... the preference is to break **after** the `||`